src/elements/Overlay/Overlay.tsx

```diff
@@ -1,16 +1,29 @@
 import { BaseHTMLElement } from '../BaseHTMLElement'
 import { css } from '../../util/taggedString'
 import { baact } from '../../../baact/baact'
+import { baatSymbol } from '../../core/BAAT'
+import { BAATEvent, HighlightElement } from '../../types'
+import { theme } from '../../theme'
+import { getBoundingBox } from '../../util/dom'
 
 const styles = css`
     :host {
         z-index: 999999;
-        width: 100%;
-        height: 100%;
-        top: 0;
-        left: 0;
+        inset: 0;
         pointer-events: none;
-        position: fixed;
+        position: absolute;
+    }
+  
+    .highlight {
+        margin: -2px;
+        background-color: rgba(0, 0, 0, 0);
+        border: 2px solid ${ theme.palette.primary };
+        pointer-events: none;
+        transition: all 0.75s ease-in;
+    }
+  
+    .blink {
+        background-color: ${ theme.palette.primaryLight };
```

Comment:
- Use a background color that at the end of the animation is not fully opaque.
- Add a light colored outline (also animated) to enhance the visibility on dark pages.
- Also animate the border to have a smoother experience?